### ISSUE TITLE
upgrade snapweb to latest version

### ISSUE THREAD

**Issue by bendschs** [jaedb/Iris#943]
### Problem to solve

Snapweb Version 6.0 is out, which solves the problem of streaming to IOS.
Would be a great addition to iris.

### Proposed solution

https://github.com/badaix/snapweb/releases

**Comment by bendschs** [comment]
now version 7.0 is out

**Comment by jaedb** [comment]
The objective to date has been to implement a light version of Snapweb into Iris itself. With Iris, you can stream from Snapcast directly in the browser; have you tried this already?

Once you've enabled Snapcast (Settings -> Services -> Snapcast) you can stream to your browser from the play bar:
<img width="299" alt="image" src="https://github.com/jaedb/Iris/assets/1711444/08db0e0a-083c-4576-a790-4d6c0a61bfd4">

**Comment by bendschs** [comment]
yes, i a m aware of that. the point is, that the currently implemented snapcast does not work in safari or ios.
badaix's snapweb had the same issue and it had been resolved in the latest version.

so my request is actually to reimplement the up to date version von snapcast/snapweb into iris.

**Comment by alanmilinovic** [comment]
Any reason why the old snapcast is in use?
